assets/js/modules/analytics-4/datastore/conversion-reporting.js

```diff
@@ -16,42 +16,35 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+
 /**
  * Internal dependencies
  */
 import API from 'googlesitekit-api';
-import { combineStores, createRegistrySelector } from 'googlesitekit-data';
+import {
+	commonActions,
+	combineStores,
+	createRegistrySelector,
+	createReducer,
+} from 'googlesitekit-data';
 import { MODULES_ANALYTICS_4 } from './constants';
 import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
+import { negateDefined } from '../../../util/negate';
 
-export const selectors = {
-	/**
-	 * Checks whether the provided conversion reporting events are available.
-	 *
-	 * @since 1.135.0
-	 *
-	 * @param {Object}               state  Data store's state.
-	 * @param {string|Array<string>} events Conversion reporting events to check.
-	 * @return {(boolean|undefined)} True if all provided custom dimensions are available, otherwise false. Undefined if available custom dimensions are not loaded yet.
-	 */
```

Comment:
Why remove the documentation here? 🤔 

EDIT: Never mind, I see it was moved and this is just the diff being strange 😄